resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -614,7 +615,6 @@ MISSING   code=1.*
 
 ## Python packages
 MISSING   jupyterlab=2.*
```

Comment:
Maybe these outputs should be updated (jupyterlab and black versions here).

Reply:
Yup, thanks!